st1 [2034][ADD] stock_picking_partner_info (#170)
qrtl/rpl-custom: stock_picking_partner_info/readme/DESCRIPTION.rst

This module does the following:

> * Display address in partner field and add phone field to the transfer order form.

Reviewer: ```suggestion
* Displays the partner address and phone in the transfer form for outgoing deliveries.
```
